pkg/bootstrap/bootstrap.go

```diff
@@ -155,10 +155,12 @@ func StartMulti(
 	if !localMode {
 		// construct multicluster watcher and client
 		clusterWatcher = watch.NewClusterWatcher(
-			ctx, manager.Options{
+			ctx,
+			manager.Options{
 				Namespace: "", // TODO (ilackarms): support configuring specific watch namespaces on remote clusters
```

Comment:
Is this todo still needed?

Reply:
we should leave the TODO here. the manager options are for the manager used to access resources on the remote cluster.
this PR only restricts the namespaces of the watched secrets on the management cluster.